src/main/java/Wall.java

```diff
@@ -0,0 +1,70 @@
+public class Wall {
```

Comment:
Same as before mate, I think you've branched off your Wall exercise work for this branch, might wanna hop on a call for 5 minutes to sort this out at some point 👍